smartsim/_core/launcher/local/local.py

```diff
@@ -34,19 +36,15 @@
 from ..stepInfo import UnmanagedStepInfo, StepInfo
 from ..stepMapping import StepMapping
 from ..taskManager import TaskManager
+from ...utils.helpers import encode_cmd
+from ...config.config import get_config
 
 logger = get_logger(__name__)
-
+cfg = get_config()
```

Comment:
Not wrong, but there is a bit of pattern (anti-pattern?) that SS uses that I'd like to stick to for now:

SS will actually make a global `CONFIG` config that can be imported here
```py
from smartsim._core.config import CONFIG
```
while the `get_config()` method actually makes a new instance. This shouldn't  make a difference as of today bc as far as I can recall the `Config` class is stateless, but even still I'd like to keep that pattern in case we do add some state in future

Reply:
Swapped to shared instance of config!